### PR TITLE
Folder hierarchy, scss setup

### LINKED ISSUE #17: Initial project config
- Use scss for styling
- Folder hierarchy

### PR DESCRIPTION
**Pull request by max-gurr**
- Installed node-sass with saved dependency
- Created folder structure
- Converted css files to scss files
- Added some basic styling

### CODE REVIEW

File: src/stylesheets/index.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import 'variables.scss';
```
Comment (yuno99825) [inline src/stylesheets/index.scss:1]: We should use @use instead of @import. @import is being phased out by SASS.

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import './App.css';
```
Comment (yuno99825) [inline src/App.js:1]: What did you use for the indentation? It doesn't look consistent - we should be using prettier/eslint

File: src/stylesheets/index.scss (hunk 1 above)
Comment (max-gurr) [reply src/stylesheets/index.scss:1]: @use only imports it into the current file

Comment (max-gurr) [reply src/stylesheets/index.scss:1]: We should be fine to use @import for this project, as it's not being deprecated for another year and a half

File: src/App.js (hunk 2 above)
Comment (yuno99825) [reply src/App.js:1]: Will leave this issue for the CI/CD team

Comment (max-gurr) [reply src/App.js:1]: I think we need a linting script, this should be a separate issue

**Review by yuno99825 — APPROVED** [review]
Everything looks good. Note that we are leaving the linting issue for later to be resolved by the CI/CD team.

**Pull request merged**